rust/cymbal/src/types/frames.rs

```diff
@@ -47,3 +48,30 @@ pub struct Frame {
     pub resolved: bool,                  // Did we manage to resolve the frame?
     pub resolve_failure: Option<String>, // If we failed to resolve the frame, why?
 }
+
+impl Frame {
+    pub fn include_in_fingerprint(&self, h: &mut Sha512) {
+        if !self.in_app {
+            return;
+        }
+        if let Some(resolved) = &self.resolved_name {
+            h.update(resolved.as_bytes());
+        } else {
+            h.update(self.mangled_name.as_bytes());
+        }
+
+        if let Some(source) = &self.source {
+            h.update(source.as_bytes());
+        }
+
+        if let Some(line) = self.line {
+            h.update(line.to_string().as_bytes());
+        }
+
+        if let Some(column) = self.column {
+            h.update(column.to_string().as_bytes());
+        }
```

Comment:
Is it necessary to include these in fingerprinting? I'd imagine they might change relatively frequently if you make a couple of attempts to fix an issue. In those cases I wouldn't want it to create multiple issues